metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/ThrottlingValues.java

```diff
@@ -0,0 +1,68 @@
+package eu.europeana.metis.core.workflow.plugins;
```

Comment:
In my point of view, I see this class as settings needed for Media Processing. The `DpsTaskSettings` (previously known as `EcloudBasePluginParameters`) is in the same package as well and is used to save the settings that are used for preparing the dps task, and it contains the `ThrottlingValues` class as a variable.
Besides that, if I were to move this class to `eu.europeana.metis.core.execution`, it will create a circular dependency between `metis-core-service` and `metis-core-common` which we should avoid. I can try to move some classes around to try to avoid this circular dependency but I believe it is best to leave as it is

Reply:
Ah, I see. Yes, you're right. It can stay in this package.